sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -740,6 +784,12 @@ export interface DeleteOperationInput {
     partitionKey?: PartitionKey;
 }
 
+// @public
+export interface EpkRange {
```

Comment:
@jcocchi This has a different name than FeedRange, should this be consistent?

Reply:
I can rename this to FeedRange.